--- origsrc/php-gtk/ext/scintilla/config.m4	2010-12-31 00:15:27.000000000 -0600
+++ src/php-gtk/ext/scintilla/config.m4	2011-01-04 19:44:59.832988300 -0600
@@ -6,12 +6,13 @@ define(gthread_required_version, 2.6.0)
 PHP_GTK_ARG_ENABLE(scintilla,for scintilla support,
 [  --enable-scintilla          Enable Scintilla support],no)
 
+PHP_REQUIRE_CXX()
+
 if test "$PHP_GTK_SCINTILLA" != "no"; then
   AC_DEFINE(HAVE_SCINTILLA, 1, [scintilla support])
   AM_PATH_GLIB_2_0(gthread_required_version,,[AC_MSG_ERROR(The scintilla extension requires GThread gthread_required_version or higher)],gthread)
   PHP_EVAL_INCLINE($GLIB_CFLAGS)
   PHP_EVAL_LIBLINE($GLIB_LIBS -lstdc++, PHP_GTK2_SHARED_LIBADD)
-  PHP_REQUIRE_CXX()
 
   scintilla_lib_srcs="libscintilla/src/AutoComplete.cxx \
 libscintilla/src/CallTip.cxx \
